ignite/contrib/handlers/tqdm_logger.py

```diff
@@ -1,3 +1,8 @@
+try:
+    import collections.abc as _collections_abc
```

Comment:
we could do 
```
from collections.abc import Mapping, Sequence 
```
it would avoid the need for `_collections_abc`

Reply:
I personally tend to avoid importing symbols that aren't packages or modules unless they're from the `typing` module, but that's just personal preference.

If we import those specifically like you have and someone did want to add typing annotations in the future, you'd get a namespace clash.

That's just personal preference though.